ts/src/base/Exchange.ts

```diff
@@ -1684,6 +1687,19 @@ export default class Exchange {
         return parseInt (convertedNumber);
     }
 
+    afterConstruct () {
+        this.generateNetworkIdToCodeMappings ();
+    }
+
+    generateNetworkIdToCodeMappings () {
```

Comment:
this name is a bit confusing, why not something like `reverseNetworkIdsObject` ?

Reply:
I think that it's always clear to name `convert_x_to_y` (mentioning `to` preposition), which is self-explanatory imo (compared to that case, as someone might as what does "reverse ids" means),  but ok as you say, i'll edis this now.

Reply:
I think that it's always clear to name `convert_x_to_y` (mentioning `to` preposition), which is self-explanatory imo (compared to that case, as someone might as what does "reverse ids" means),  but ok as you say, i'll edis this now.